### ISSUE TITLE
NTR: History of gastric bypass (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7881]
New term request History of gastric bypass (jadhav_091115193556): Stomach  UMLS: 0017125

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000179
- label: History of gastric bypass surgery
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:31985950
- def: History of having received gastric bypass surgery. This is a procedure that reduces the functional volume of the stomach and is used to treat morbid obesity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7881
- synonyms:
